setup.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#! /usr/bin/env python
```

Comment:
We need to set all shebangs back to `#! /usr/bin/env python`
Almost all scripts (either on sandbox/ or scripts/) need to be changed.